Python/getargs.c

```diff
@@ -437,6 +437,27 @@ seterror(Py_ssize_t iarg, const char *msg, int *levels, const char *fname,
 }
 
 
+static const char *
+convertitems(PyObject *arg, const char **p_format, va_list *p_va, int flags,
```

Comment:
`args` is always a tuple here, so `convertitems` and `converttuple` have counterintuitive names.